Cargo.toml

```diff
@@ -111,6 +112,7 @@ displaydoc = { version = "0.2.4", default-features = false }
 
 cfg-if = "1.0.0"
 derive_more = { version = "0.99.17", default-features = false }
+parse-display = "0.8.2"
```

Comment:
I don't think this dependency is worth it. We can already derive `Display` with `derive_more` and `FromStr` with `strum::EnumString`

Reply:
Removed (although it is still used by FFI derive)

Reply:
would you please remove from there also?

Reply:
In order to not bloat this PR any further, will create a separate small ticket for it.